layers/lldp.go

```diff
@@ -240,7 +240,7 @@ type LinkLayerDiscoveryInfo struct {
 	Unknown         []LinkLayerDiscoveryValue // undecoded TLVs
 }
 
-/// IEEE 802.1 TLV Subtypes
+// / IEEE 802.1 TLV Subtypes
```

Comment:
spurious slash?

Reply:
this is an automate output of `go fmt`. not really keen to change it since it'll get reverted back to this on save anyway. I'll take a look in the context of that particular line though.